app/src/main/java/pl/futurecollars/invoicing/model/InvoiceEntry.java

```diff
@@ -0,0 +1,20 @@
+package pl.futurecollars.invoicing.model;
+
+import java.math.BigDecimal;
+import lombok.Data;
+
+@Data
+public class InvoiceEntry {
+
+    private final String description;
+    private final BigDecimal price;
+    private final Vat vatRate;
+    private final BigDecimal vatValue;
+
+    public InvoiceEntry(String description, BigDecimal price, Vat vatRate) {
+        this.description = description;
+        this.price = price;
+        this.vatRate = vatRate;
+        this.vatValue = price.multiply(BigDecimal.valueOf(vatRate.rate));
```

Comment:
This line will not work as you desire. 
Check this out:
https://ideone.com/3liEV9